filterparser.go

```diff
@@ -361,7 +361,7 @@ func (p *FilterParser) parseOperatorAndValue() (parserToken, interface{}, error)
 	}
 
 	if _, ok := p.config.unsupportedComparators[operator]; ok {
-		return parserTokenILLEGAL, nil, fmt.Errorf("unsupported comparator:  %s", tokensToOperator[operator])
+		return parserTokenILLEGAL, nil, fmt.Errorf("unsupported comparator: %s", tokensToOperator[operator])
```

Comment:
wasn't that part of a previous pr?

Reply:
Oops, I realized I added two spaces in the last PR and I fixed it in this PR instead of the last one 🙈 

We want this fix! Sorry for the confusion.

Reply:
This is fixed now ✅